src/relay/backend/te_compiler.cc

```diff
@@ -566,100 +566,128 @@ class LowerTensorExprMutator : public DeviceAwareExprMutator {
         return itr->second;
       }
     } else if (const auto* function_node = expr.as<FunctionNode>()) {
-      if (!function_node->HasNonzeroAttr(attr::kPrimitive)) {
-        // Not marked as primitive by FuseOps.
-        return {};
-      }
-      if (const auto* call_node = function_node->body.as<CallNode>()) {
-        if (call_node->op == debug_op_) {
-          // Debug 'primitives' are not lowered.
-          return {};
+      if (function_node->HasNonzeroAttr(attr::kExtern)) {
```

Comment:
does this functionality need test?

Reply:
Everything needs testing, but there's nothing to build on so we keep kicking the can ahead of us.How about as a compromise I put some love into the comments so we at least know what needs to be tested?